INSTALL.md

```diff
@@ -121,7 +121,55 @@ For bare-metal installations, the full instructions for setting up PostgreSQL, T
     LATEST_VERSION=$(curl -s https://api.github.com/repos/timescale/promscale/releases/latest | grep "tag_name" | cut -d'"' -f4)
     curl -L -o promscale "https://github.com/timescale/promscale/releases/download/${LATEST_VERSION}/promscale_${LATEST_VERSION}_Linux_x86_64"
     chmod +x promscale
-    ./promscale --db-name promscale --db-password promscale --db-user promscale --db-ssl-mode allow --install-extensions
+    ./promscale --db.name promscale --db.password promscale --db.user promscale --db.ssl-mode allow --startup.install-extensions --startup.upgrade-prerelease-extensions
+    ```
+## Compile From Source on MacOS(Darwin, ARM64)
+
+For bare-metal installations, the full instructions for setting up PostgreSQL, TimescaleDB, and the Promscale Extension are:
+
+1) Follow [this doc]([url](https://docs.timescale.com/install/latest/self-hosted/installation-macos/#install-self-hosted-timescaledb-using-homebrew)) and install Timescale(which also install Postgres)
+    ```bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    ```
+1) Restart Postgres after executing `timescaledb-tune` script
+    ```bash
+    pg_ctl -D /opt/homebrew/var/postgres restart
+    ```
+1) [Install rust](https://www.rust-lang.org/tools/install).
+    ```bash
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    source $HOME/.cargo/env
+    ```
+1) Install the PGX framework
+    ```bash
+    cargo install cargo-pgx --git https://github.com/timescale/pgx --branch promscale-staging
+    ```
+1) Initialize the PGX framework using the PostgreSQL 14 installation
+    ```bash
+    cargo pgx init --pg14=$(which pg_config)
+    ```
+1) Download this repo and change directory into it
+    ```bash
+    git clone https://github.com/timescale/promscale_extension
+    cd promscale_extension
+    git checkout master
+    ```
+1) Compile and install
+    ```bash
+    make package
+    sudo cp -r ./target/release/promscale-pg14/* /
```

Comment:
Does `sudo make install` not work here?

Reply:
No, `--recursive` is not a recognised option in darwin.

Reply:
You are hitting https://github.com/timescale/promscale_extension/pull/391. 

> I tried with make install and it kind of work.

Probably you installed `cp` from GNU utils through brew?

Reply:
probably, I replace a lot of the default macos tools by the gnu counterparts installed via brew